webactivity.py

```diff
@@ -28,7 +28,7 @@
     gi.require_version('WebKit2', '4.0')
```

Comment:
The 4.0 API for webkit2gtk uses libsoup2.4. The 4.1 API is identical except it uses libsoup3.

Since it's not possible to use libsoup2.4 and libsoup3 in the same process, this would need to be changed to 4.1

Reply:
Thanks, amended.